plugin.video.youtube/resources/lib/youtube/client/login_client.py

```diff
@@ -19,83 +35,57 @@ class LoginClient(object):
         'youtube-for-kodi-quota': {
             'token-allowed': False,
             'system': 'All',
-            'key': 'AIzaSyA7v1QOHz8Q4my5J8uGSpr0zRrntRjnMmk',
-            'id': '597640352045-7um2gr1v5rgobm5bf07ebesm3er48286.apps.googleusercontent.com',
-            'secret': 'VmyQ12KkJ_N3yegu4Y-VGCXd'
+            'key': '%s' % api_key,
```

Comment:
Ok Would it be ok if we added this to the next release or would you prefer
us to do it before proceeding forward?

Thanks for your help!

Jeff

On Mon, May 23, 2016 at 11:25 AM, Philipp Temminghoff <
notifications@github.com> wrote:

> In plugin.video.youtube/resources/lib/youtube/client/login_client.py
> <https://github.com/xbmc/repo-plugins/pull/480#discussion_r64240401>:
>
> > @@ -19,83 +35,57 @@ class LoginClient(object):
> >          'youtube-for-kodi-quota': {
> >              'token-allowed': False,
> >              'system': 'All',
> > -            'key': 'AIzaSyA7v1QOHz8Q4my5J8uGSpr0zRrntRjnMmk',
> > -            'id': '597640352045-7um2gr1v5rgobm5bf07ebesm3er48286.apps.googleusercontent.com',
> > -            'secret': 'VmyQ12KkJ_N3yegu4Y-VGCXd'
> > +            'key': '%s' % api_key,
>
> just api_key instead of '%s' % api_key might be more readable. ;)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/repo-plugins/pull/480/files/5ab7db1a24d3b6ff8e2f8a4d594407523dcf890a#r64240401>
>